main.py

```diff
@@ -0,0 +1,1605 @@
+    mqtt_info.tag_configure("align", justify='center')      #configure tags
+    mqtt_info.tag_configure("bold", font=(None, 10, BOLD))
+    mqtt_info.tag_configure("spacing", font=(None, 3,))
+
+    mqtt_endpoint_text = 'MQTT Endpoint:'
+    mqtt_topic_prefix_text = 'MQTT Topic Prefix:'
+    mqtt_client_id_text = 'MQTT Client ID:'
+
+    mqtt_info.insert(END, mqtt_endpoint_text + '\n', "bold")
+    mqtt_info.insert(END, mqtt_endpoint + '\n')
+    mqtt_info.insert(END, '\n', "spacing")
+    mqtt_info.insert(END, mqtt_topic_prefix_text + '\n', "bold")
+    mqtt_info.insert(END, mqtt_topic_prefix + '\n')
+    mqtt_info.insert(END, '\n', "spacing")
+    mqtt_info.insert(END, mqtt_client_id_text + '\n', "bold")
+    mqtt_info.insert(END, client_id)
+    
+    mqtt_info.tag_add("align", 1.0, END)
+    mqtt_info['state'] = DISABLED
+
+    #(0,4) show account type
+    account_type_frame = Frame(frame, borderwidth=2, relief=RAISED, background='#298b9e')
+    account_type_frame.grid(column=0, row=6, padx=(30,20))
+
+    account_type_text = Text(account_type_frame, background='#298b9e', foreground='white', width=25, height=2, wrap=CHAR)
+    account_type_text.config(highlightthickness=0, borderwidth=0, font=(myFont, 10), spacing1=5, spacing2=2, spacing3=2)
+    account_type_text.pack(padx=15, pady=7, fill=BOTH, expand=TRUE)
+    account_type_text.tag_configure("align", justify='center')  #configure tags
+    account_type_text.tag_configure("bold", font=(None, 10, BOLD))
+
+    account_type_label = 'Account Type:'
+
+    account_type_text.insert(END, account_type_label + '\n', "bold")
+    account_type_text.insert(END, account_type)
+    account_type_text.tag_add("align", 1.0, END)    #center all
+    account_type_text['state'] = DISABLED 
+
+    #(0,7) bottom left, log off button
+    log_off_frame = Frame(frame, background=light_grey)
+    log_off_frame.grid(column=0, row=7, padx=10, pady=15, sticky=W+E+S)
+
+    log_off = Button(log_off_frame, text="Log Off", command=restartPopup)
+    log_off.config(activebackground='#9ed3e8', activeforeground='white', background='#505f63', foreground='white')
+    log_off.pack(pady=(0,10), fill=BOTH, expand=TRUE)
+    button_config(log_off)
+
+    for widget in frame.winfo_children():
+        widget.grid(padx=5, pady=1)
+    return frame
+
+def main_screen():
+    '''Main Screen Configuration'''  
+    root.attributes('-alpha', 1)    #turn off transparency for main page
+    #build two frames for main window
+    root.columnconfigure(0, weight=1)
+    root.columnconfigure(1, weight=6)
+    root.rowconfigure(0, weight=1)
+
+    left_frame = create_left_frame(root)
+    left_frame.config(background=light_grey)
+    left_frame.grid(column=0, row=0, sticky=W+E+N+S)
+
+    right_frame = create_right_frame(root)
+    right_frame.config(background=light_grey)
+    right_frame.grid(column=1, row=0, sticky=W+E+N+S)
+
+def edit_config_file(flag, data): #store new values into file
+    if flag == 1:   #store into API
```

Comment:
This is certainly one way to do this; there are many ways of course.  Another could be to replace the numeric flag values 1-3 with Enum values by name: https://docs.python.org/3.11/howto/enum.html.